graphite/graphite_tcp.go

```diff
@@ -0,0 +1,63 @@
+package graphite
+
+import (
+	"bufio"
+	"io"
+	"log"
+	"net"
+)
+
+// TcpGraphiteServer processes Graphite data received over TCP connections.
+type TcpGraphiteServer struct {
+	GraphiteServer
+}
```

Comment:
Can you rename this to `TCPServer`? "Graphite" adds a stutter when used with the package name.